index.md

```diff
@@ -10,6 +10,16 @@ title: "Ruby Unconf Hamburg 2019"
 Ruby Unconf Hamburg 2019 will happen on 25th and 26th of May 2019 at the HAW Hamburg, DMI Campus Finkenau.
 More information will follow soon.
 
+Blog posts:
```

Comment:
It would be awesome if you can move the blog posts into a dedicated section.

I have no strong opinion where to add the blog section exactly, I think I would add it somewhere underneath the tickets section.

Reply:
we may should change this whole section when releasing the blog. I will think about it what we can change. If you have a idea you're welcome to contribute (you can do it directly within this branch)

Reply:
on a second thought I would just leave it as it is for now and iterate the structure of the first section in a follow up MR when our blog is online

> At the same time I am a fan of:
>>    Make it work. Make it pretty.

Reply:
I am not a fan of keeping the blog entries above the tickets section, because the blog should not add any friction that might harm ticket sales.

You can style the list of blog articles hower you think is best.

Reply:
IMHO news on the top is more appealing than having the ticket widget. Also when the navigation is finished there is a direct link to Tito on the top

Reply:
I still think tickets is more important than news (I will happily elaborate on the reasons in person). And the blog should be part of the main menu as well. Have a look at https://2019.jsconf.eu/ for example, how they handle blog / news.